src/app/core/data/request.service.ts

```diff
@@ -66,9 +67,10 @@ export class RequestService {
       .flatMap((uuid: string) => this.getByUUID(uuid));
   }
 
-  configure<T extends CacheableObject>(request: RestRequest): void {
-    if (request.method !== RestRequestMethod.Get || !this.isCachedOrPending(request)) {
-      this.dispatchRequest(request);
+  // TODO to review "overrideRequest" param when https://github.com/DSpace/dspace-angular/issues/217 will be fixed
```

Comment:
I'd pull the `trackRequestsOnTheirWayToTheStore` if out of `dispatchRequest`, rename `overrideRequest` to `forceBypassCache` to make it clearer what it does, and add the `ResponseCacheRemoveAction` you proposed in #217:
 
```typescript 
configure<T extends CacheableObject>(request: RestRequest, forceBypassCache: boolean = false): void {
  const isGetRequest = request.method === RestRequestMethod.Get;
  if (forceBypassCache || !isGetRequest || !this.isCachedOrPending(request)) {
    this.dispatchRequest(request);
    if (isGetRequest && !forceBypassCache) {
      this.trackRequestsOnTheirWayToTheStore(request);
    }
  }
}
private dispatchRequest(request: RestRequest) {
  this.store.dispatch(new ResponseCacheRemoveAction(request.href));
  this.store.dispatch(new RequestConfigureAction(request));
  this.store.dispatch(new RequestExecuteAction(request.uuid));
}
```

The `ResponseCacheRemoveAction` will likely make it unnecessary for you to use `forceBypassCache` in this PR, but I still think it's a good option to have. We may need it in the future, so I'd leave it in.

Reply:
unluckily this cause some problem with the MyDSpace, because the same rest call are made several times. I've found a solution that we don't need to include in this PR